spec/buho_cfdi/nodes/tax_transferred_spec.rb

```diff
@@ -1,19 +1,27 @@
-RSpec.describe Nodes::TaxTransferred, type: :model do
-  it { should respond_to(:base) }
-  it { should respond_to(:tax) }
-  it { should respond_to(:factor_type) }
-  it { should respond_to(:rate_or_fee) }
-  it { should respond_to(:import) }
-
-  context 'required params' do
-    it { should validate_presence_of(:base) }
-    it { should validate_presence_of(:tax) }
-    it { should validate_presence_of(:factor_type) }
+RSpec.describe ::Nodes::TaxTransferred, type: :model do
+  context 'when factor_type is not Exento' do
+    let (:node) { ::Nodes::TaxTransferred.new({tax: '001', factor_type: 'Tasa', base: '100', rate_or_fee: '0.1000', import: '100'}) }
+    it 'has all values' do
+      expect(node.valid?).to eq(true)
```

Comment:
```suggestion
      expect(node).to be_valid
```

Reply:
All my changes are in 14ea7dcb31609090f517cdcff8298a0f98c7dd5c